pkg/metrics/chaos-controller-manager.go

```diff
@@ -83,65 +79,58 @@ func NewChaosCollector(store cache.Cache, registerer prometheus.Registerer) *Cha
 			Name: "chaos_mesh_injections_total",
 			Help: "Total number of sidecar injections performed on the webhook",
 		}, []string{"namespace", "config"}),
+		reconcileDuration: prometheus.NewHistogramVec(prometheus.HistogramOpts{
```

Comment:
As we have to decide to use the origin metrics provided by controller-runtime, I think  we could also remove `reconcileDuration`.

Reply:
Oh, is ` workqueue_queue_duration_seconds` corresponding to reconcile duration metric? I misunderstood it before.

Reply:
No, here is a metrics called `controller_runtime_reconcile_time_seconds`. :)

Reply:
You could find it at pkg/internal/controller/metrics/metrics.go:44 from controller-runtime.

Reply:
And it's also much better to use `time.Duration` instead of `time.Time` when you collect data for the "timer" metrics. :D

Reply:
Ah, sorry, I didn't see this before.